intelmq/bots/BOTS

```diff
@@ -728,6 +728,18 @@
                 "port": "<port>"
             }
         },
+        "Redis": {
+            "description": "Redis is the bot responsible to send events to a Redis server.",
+            "module": "intelmq.bots.outputs.redis.output",
+            "parameters": {
+                "redis_server_ip": "127.0.0.1",
```

Comment:
The parameters aren't sorted, thats why the build is failing. Sorry for being pedantic, but otherwise this file would be a mess (We have been there already).